runtime/core/src/entities/entity.js

```diff
@@ -62,6 +61,9 @@ class Entity {
         value: {}
       })
 
+    if (!('_created' in value))
+      value._created = Date.now()
+
     if (this.#state !== undefined) {
       value._updated = Date.now()
```

Comment:
Any reasons why we don't want to use database level functions for timestamps? MongoDb provides [$currentDate](https://www.mongodb.com/docs/manual/reference/operator/update/currentDate/) for that

Reply:
1. To avoid implementing this in all storage connectors.
2. The source of time in the distributed (and especially in geo-distributed) systems is a complicated question that we'll discuss eventually. Neither this, nor database time are the correct implementation.